docs/guides/update-guide/810-to-820.md

```diff
@@ -33,5 +33,5 @@ This may change the performance characteristics. If you have tuned your system f
 - New configuration is required to enable Tasklist Process Page:
   - `CAMUNDA_TASKLIST_IDENTITY_BASEURL` environment variable should be configured to your Identity host
   - `CAMUNDA_TASKLIST_IDENTITY_RESOURCE_PERMISSIONS_ENABLED` environment variable should be set to true or false (default is false) and configures Tasklist to use Authorizations coming from Identity.
-    - This should be in alignment with Identity configuration as it should just be enabled if it is also enabled in Identity
+    - To enable this variable in Tasklist, Resource authorizations must be [enabled in Identity](../../identity/user-guide/authorizations/managing-resource-authorizations/).
```

Comment:
We will document how to enable the feature in the concepts section, so it is easier discovered by users.

@christinaausley a PR with the new page is on the way.

```suggestion
    - To enable this variable in Tasklist, Resource authorizations must be [enabled in Identity](../../concepts/access-control/resource-authorizations/).
```

Reply:
@christinaausley and @dlavrenuek should we hold this PR until the new page is created or should we merge it?

Reply:
@dlavrenuek @marcosgvieira Up to you -- if you want to go ahead and get this PR out the door we can always start fresh with a new one.

Reply:
@marcosgvieira we will have cross-references f.e. from [this PR](https://github.com/camunda/camunda-platform-docs/pull/1897), so you can go ahead and merge if the only problem is the missing page